templates/prometheus.yml.j2

```diff
@@ -34,4 +34,19 @@ scrape_configs:
 
   - job_name: 'node'
     static_configs:
-      - targets: ['{{ NODEEXPORTER_SERVER_NAME }}:9100']
\ No newline at end of file
+      - targets: ['{{ NODEEXPORTER_SERVER_NAME }}:9100']
+
+  - job_name: 'blackbox'
+    metrics_path: /probe
+    params:
+        module: [http_2xx]
+    static_configs:
+        - targets:
+            - https://www.alliancegenome.org/api/search_autocomplete?q=p
```

Comment:
Also its not clear which container is getting restarted when this link goes down.

Reply:
The AWS Lambda script is in the "agr_monitoring" repository. This just triggers an SNS message sent to AWS Lambda.

Reply:
Ahh ok.